infrastructure/docker_image/Dockerfile.template

```diff
@@ -11,4 +11,4 @@ COPY task_script.sh .
 ENV NMAP_TARGET_STRING=127.0.0.1
 ENV RESULTS_BUCKET=${results_bucket_arn}
 
-ENTRYPOINT [ "sh", "task_script.sh", "--resolve-all", "--script", "ssl-enum-ciphers,vulners,ssl-cert", "-A", "-v" ]
\ No newline at end of file
+ENTRYPOINT [ "sh", "task_script.sh", "--resolve-all", "--script", "ssl-enum-ciphers,vulners,ssl-cert", "-A", "-v", "-sT", "-p0-65535", "-T4"]
```

Comment:
We now have way too many arguments to not comment on them. We should explain exactly what options we are passing and why